include/openenclave/bits/sgx/sgxproperties.h

```diff
@@ -83,6 +87,35 @@ typedef struct _oe_sgx_enclave_properties
     (OE_SGX_FLAGS_MODE64BIT | (ALLOW_DEBUG ? OE_SGX_FLAGS_DEBUG : 0) | \
      (REQUIRE_KSS ? OE_SGX_FLAGS_KSS : 0))
 
+#define OE_SE_PAGE_SIZE 0x1000
```

Comment:
What does `SE` stand for?

Reply:
Not sure what SE stands for actually (maybe security enhanced, like in SELinux). I had taken the constant from intel/linux-sgx. Renamed to OE_SGX_PAGE_SIZE.